ctapipe/io/__init__.py

```diff
@@ -18,7 +14,5 @@
     'TableWriter',
     'TableReader',
     'EventSeeker',
-    'EventSourceFactory',
     'EventSource',
-    'event_source'
```

Comment:
is it necessary to remove event_source here? It is nice to be able to just type `from ctapipe.io import event_source`, since the fact that there even exists a factory can be hidden to the user. Having to explicitly import it from the sub-package is a bit cumbersome.

Reply:
What do you mean by "With this plugin scheme"? Do you know a different scheme which would work?

Reply:
I don't, I just didn't want to rule out the possibility. I think this is the right plugin scheme to use, and we should change the Factory such that it works better with it.